z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-installer-rhcos/tasks/main.yaml

```diff
@@ -11,9 +11,19 @@
 - name: 'Import common yaml'
   import_tasks: common.yaml
 
+- name: Verify openshift minor version
+  shell: |
+    version=$(curl -s https://mirror.openshift.com/pub/openshift-v4/{{ ansible_architecture }}/dependencies/rhcos/{{ openshift_version }}/{{ openshift_version }}.{{ openshift_minor_version }}/sha256sum.txt | grep rhcos | wc -l)
+    if [ $version -eq "0" ] || [ "{{ openshift_minor_version }}" -eq "latest" ]; then
+      echo $(curl -s https://mirror.openshift.com/pub/openshift-v4/{{ ansible_architecture }}/dependencies/rhcos/{{ openshift_version }}/latest/sha256sum.txt | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+" | head -n1)
+    else
+      echo "{{ openshift_version }}.{{ openshift_minor_version }}"
+    fi
+  register: icic_ocp_version
+
 - name: 'Set fact for openshift client'
   set_fact:
-    openshift_rhcos_url: "https://mirror.openshift.com/pub/openshift-v4/{{ ansible_architecture }}/dependencies/rhcos/{{ openshift_version }}/{{ openshift_version }}.{{ openshift_minor_version }}"
+    openshift_rhcos_url: "https://mirror.openshift.com/pub/openshift-v4/{{ ansible_architecture }}/dependencies/rhcos/{{ openshift_version }}/{{ icic_ocp_version.stdout }}"
```

Comment:
Why we need `set fact for openshift client `here since we have done it in `configure-installer-client`

Reply:
give the wrong task name, already change to openshift rhcos url.